src/app/content/highlights/utils/paginationUtils.spec.ts

```diff
@@ -73,17 +73,14 @@ describe('filterCountsPerSourceByLocationFilter', () => {
   it('filters by pages outside chapter', () => {
     expect(filterCountsPerSourceByLocationFilter(
       new Map([
-        ['chapter1', findArchiveTreeNodeById(treeWithUnits, 'chapter1')!],
-        ['preface', findArchiveTreeNodeById(treeWithUnits, 'preface')!]]
```

Comment:
Why was this removed? I'm not sure why does `treeWithUnits` not contain preface.

Reply:
you are right, I should have just move preface outside of a unit - I've pushed a change for this